sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/models/area.json

```diff
@@ -0,0 +1,28 @@
+{
+    "@id": "dtmi:billmanh:area;1",
```

Comment:
Could we select more generic IDs for the various models?

Reply:
Sure I can do that.

Reply:
Changed it to `mymodels`